backend/vc-certification-service/src/controllers/certificate.controller.js

```
@@ -1,16 +1,26 @@
 const sunbirdRegistryService = require('../services/sunbird.service')
 const certifyConstants = require('../configs/constants');
 const {validationResult} = require('express-validator');
+const validationService = require('../services/validation.service')
 async function createCertificate(req, res) {
     try {
         const entityType = req.params.entityType;
         const token = req.header("Authorization");
         console.log("EntityType: ", entityType);
-        const certificateAddResponse = await sunbirdRegistryService.createCertificate(req.body, entityType, token)
-        res.status(200).json({
+        const isValidPayload = validationService.validateCertificateInput(req);
```

instead of returning error in message format, can we throw error from validateCertificateInput. In that way, we wont need if else cases and let catch block handle the error objects thrown
error objects can contain statusCode of 400 and then appropriate message.
eg:
```
err = {
    response: {
          status: 400,
          data: "some error"
    }
}